### PR TITLE
Create and render ball on screen

### PR DESCRIPTION
**Pull request by IvarForkbeard**
I think this ball is ready for the big time.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -9,6 +10,7 @@ if (!c) {
 }
 
 const paddles = [new Paddle({ x: 10, y: 60, width: 20, height: 50, color: 'teal' })];
+const ball = [new Ball({ x: 100, y: 60, radius: 12, color: 'yellow' })];
```
Comment (eric-decompiled) [inline main.js]:
```suggestion
const ball = new Ball({ x: 100, y: 60, radius: 12, color: 'yellow' });
```

or

```suggestion
const balls = [new Ball({ x: 100, y: 60, radius: 12, color: 'yellow' })];
```

**Comment by eric-decompiled** [comment]
also you should try to merge the latest change in `main` into this branch first.

from CLI

`git fetch`
`git merge origin/main`

Git kraken should have an option for it to.

This will let you see what the actual code will be in `main`. Not bein up to date is risky on merges if your code isn't up to date with the target branch.

File: main.js (hunk 1 above)
Comment (IvarForkbeard) [reply main.js]: Can you explain these changes?  Why do paddles get [] but ball doesn't?  and why change ball to balls?  Are we making the game expandable to have a multiball function?

Comment (eric-decompiled) [reply main.js]: Its kind of nitty, but the `paddles` variables is plural, which helps suggest its an array.

With ball you have it as an array, but called it a singular

Comment (eric-decompiled) [reply main.js]: If you only have a single ball you don't need an array and it will make accessing it easier

**Review by eric-decompiled — APPROVED** [review]

**Review by eric-decompiled — APPROVED** [review]
lgtm

**Pull request merged**